test/testapps/BasicTestApp/RouterTest/Links.cshtml

```diff
@@ -1,5 +1,5 @@
-﻿@inject Microsoft.AspNetCore.Blazor.Services.IUriHelper uriHelper
-
+@page "/Links"
```

Comment:
Do we need the `@page` directive here? I thought this was only used as a component, not as a routable page in itself.

Reply:
Maybe not. I added it just by inertia